app/models/order_item.rb

```diff
@@ -0,0 +1,9 @@
+class OrderItem < ApplicationRecord
+	belongs_to :order, dependent: :destroy
+	belongs_to :item
+	has_one    :line_item
+	validates  :quantity, :numaricality => {greater_than: 0}
+	validates  :Item_id, :order_id, presence: true
```

Comment:
similarly add validation for price

Reply:
Item_id => item_id

Reply:
done
changed Item_id to item_id and also place to price validation